lib/docker/template/cache.rb

```diff
@@ -17,12 +17,10 @@ def context(builder, context)
         builder.repo.cache_dir.rm_rf
         $stderr.puts Simple::Ansi.yellow(format("Copying context for %s", builder.repo))
         cache_dir = builder.repo.cache_dir
-        cache_dir.parent.mkdir_p
+        cache_dir.mkdir_p
 
         readme(builder)
-        context.cp_r(cache_dir.tap(
-          &:rm_rf
-        ))
+        context.join(".").cp_r(cache_dir)
```

Comment:
That happened because you stopped creating the parent and started creating the folder, this moved the folder to the folder you created rather than moving the folder to the location we wanted it at.

I know how cp_r works :wink: I Ruby and System for a living.